includes/Plugin.php

```diff
@@ -57,17 +59,181 @@ public function register() {
 		add_action( 'rest_api_init', [ $embed_controller, 'register_routes' ] );
 
 		// Dashboard.
-
 		$dashboard = new Dashboard();
 		add_action( 'init', [ $dashboard, 'init' ] );
 
 		// Migrations.
-
 		$database_upgrader = new Database_Upgrader();
 		add_action( 'admin_init', [ $database_upgrader, 'init' ] );
 
 		// Gutenberg Blocks.
 		$embed_block = new Embed_Block();
 		add_action( 'init', [ $embed_block, 'init' ] );
+
+		// Everything else.
+		add_filter( 'admin_body_class', [ __CLASS__, 'admin_body_class' ], 99 );
```

Comment:
Should these live here? Or a new class

Reply:
A new `Admin` class maybe?

Ideally we take the time at some point to properly refactor the PHP parts... Let's open an issue to discuss this more broadly?

Reply:
> A new Admin class maybe? 
Makes sense for me